pymc4/distributions/continuous.py

```diff
@@ -1270,32 +1149,26 @@ class Uniform(BoundedContinuousDistribution):
 
     Parameters
     ----------
-    lower : float|tensor
+    low : float|tensor
         Lower limit.
-    upper : float|tensor
+    high : float|tensor
         Upper limit.
-
-    Developer Notes
-    ----------------
-    Parameter mappings to TensorFlow Probability are as follows:
-
-    - lower: low
-    - upper: high
     """
 
-    def __init__(self, name, lower, upper, **kwargs):
-        super().__init__(name, lower=lower, upper=upper, **kwargs)
+    def __init__(self, name, low, high, **kwargs):
+        super().__init__(name, low=low, high=high, **kwargs)
 
     @staticmethod
     def _init_distribution(conditions):
-        lower, upper = conditions["lower"], conditions["upper"]
-        return tfd.Uniform(low=lower, high=upper)
+        low, high = conditions["low"], conditions["high"]
+        return tfd.Uniform(low=low, high=high)
 
+    # FIXME should we rename this functions as well?
     def lower_limit(self):
-        return self.conditions["lower"]
+        return self.conditions["low"]
```

Comment:
Should this function be called `lower_limit` or just `lower`?

Reply:
We can keep this naming, otherwise `lower` and `upper` alone might be confusing as some Distribution might have them as kwargs.

Reply:
If we bet bound to tfp, we better use domain objects from there

Reply:
I'm not really sure why these functions exist: e.g. `Normal` doesn't have equivalent functions that return the same distribution parameters. Maybe I'm missing something, but is it necessary to have these functions at all?